code/_globalvars/database.dm

```diff
@@ -1,19 +0,0 @@
-	// MySQL configuration
-
-GLOBAL_VAR_INIT(sqladdress, "localhost")
-GLOBAL_PROTECT(sqladdress)
-GLOBAL_VAR_INIT(sqlport, "3306")
-GLOBAL_PROTECT(sqlport)
-GLOBAL_VAR_INIT(sqlfdbkdb, "test")
-GLOBAL_PROTECT(sqlfdbkdb)

```

Comment:
I just wanted to comment that this variable name is so cryptic if you don't already know what it is, it's hilarious.
Not asking you to change it, just a silly thing.